app/src/main/java/io/appium/uiautomator2/model/settings/Settings.java

```diff
@@ -23,6 +23,7 @@ public enum Settings {
     ELEMENT_RESPONSE_ATTRIBUTES(new ElementResponseAttributes()),
     ENABLE_MULTI_WINDOWS(new EnableMultiWindows()),
     ENABLE_NOTIFICATION_LISTENER(new EnableNotificationListener()),
+    ENABLE_TOPMOST_WINDOW_FROM_ACTIVE_PACKAGE(new EnableTopmostWindowFromActivePackage()),
```

Comment:
Please add setting description into the README

Reply:
Excuse me, which README are referring to?

Reply:
I meant https://github.com/appium/appium-uiautomator2-driver/blob/master/README.md, but you are right, this will be the next step after this PR is merged

Reply:
Thank you for telling me that. Let me handle it after this PR.